tests/architecture/Test.xml

```diff
@@ -1,5 +1,6 @@
 <?xml version="1.0" encoding="UTF-8" ?>
-<rules>
+<rules xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+       xsi:schemaLocation="https://raw.githubusercontent.com/faguaron/phpat/YAML/src/Test/Test.xsd">
```

Comment:
this should be changed to *.../carlosas/phpat/master/...*
in a future I will search for a better way to do it so it's not coupled to the github path